warehouse/models/intermediate/payments/int_payments__refunds_deduped.sql

```diff
@@ -0,0 +1,169 @@
+{{ config(materialized = "table") }}
+
+WITH
+
+micropayments_table_refunds AS (
+    SELECT
+
+        aggregation_id,
+        micropayment_id,
+        participant_id,
+        customer_id,
+        ABS(charge_amount) AS proposed_amount,
+        EXTRACT(DATE FROM transaction_time) AS transaction_date,
+        aggregation_id AS coalesced_id,
```

Comment:
I think we also need to preserve `aggregation_id` as itself for joins later (in the `_to_aggregations` model, we group by `aggregation_id` so we need that column populated for these rows)

Reply:
My bad it's a few rows up